### ISSUE TITLE
Enable screen reader users to "hear" that the text is "bold" (highlighting the person that was identified by the search and clicked to)

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/FreeCENMigration#1126]
E0  U9 SS6 C6: P21
A span with a class of accessibility (Alison to expand this story)

**Comment by DeniseColbert** [comment]
@AlOneill to add some info to the story about accessibility code for this.

**Comment by AlOneill** [comment]
I suggest we precede the name with `<span class="accessibility">This is the person found in your search</span>`

There are a few more issues with this page:
* neither table has a `caption`: I suggest something along the lines of `<caption class="accessibility"></caption>` and 
* the text "If you believe there to be an error in this transcription then please report (link opens in a new tab) it to our data manager." is marked up as a level-4 heading

**Comment by AlOneill** [comment]
I suggest we follow the name with `<span class="accessibility">the person found in your search</span>`

Neither table has a `caption` to explain the contents. I suggest _something like_ `<caption class="accessibility">Reference details for this record</caption>` for the first and for the second, Details of each person in the record

The text "If you believe there to be an error in this transcription then please **report (link opens in a new tab) it** to our data manager." is marked up as a level-4 heading. It is not a heading — I think this was done to get an increase in text size rather than to confer meaning! And would "it" be better immediately after "report"?

**Comment by AlOneill** [comment]
On further thought — the link-text would be better with some more context, so I suggest the following (the link-text is in square brackets) —
"If you believe there to be an error in this transcription then please [report it to our data manager (link opens in a new tab)].

**Comment by Captainkirkdawson** [comment]
Changes applied to test3

**Comment by AlOneill** [comment]
@Captainkirkdawson Ah. I must've explained that badly. Apologies. I'll tweak the markup and email some HTML snippets.

**Comment by AlOneill** [comment]
@Captainkirkdawson 
The first table is good. Thank you.

The `span` between the tables is in the wrong place: it has to identify a specific person in the second table, so it needs to go immediately before their surname value, in the same cell:
`<td class="weight--semibold"><span class="accessibility">the person found in your search</span>SURNAME value</td>`

The `thead` of the second table is empty: I would expect it to contain the column headers — unless there is a particular reason for not putting them there? The row containing the headers is currently in the first `tbody`.

The `caption` should be the first child of a `table` to be valid.

**So**, suggest the empty `thead` is removed, thus making the caption the first child of the table. Then change first `tbody` tags to `thead`.

The "If you believe …" para (TY!) has a typo: remove second 'please', from the link-text,  making 'report' the first word (easy to identify in a list).

I hope that clarifies things and is workable.

**Comment by Captainkirkdawson** [comment]
Sorry @AlOneill. Most of your requests have been adopted. 1) Positioning of the caption in the second table and 2) removing a please. 3) putting the span on the first element required a bit more programming because of the way the values are placed in the line which is why I had placed it on the the full line.
There is no empty `thead` in the second table. It does have all the column headers within it.

**Comment by AlOneill** [comment]
@Captainkirkdawson Second table now good. Thank you.

You deleted the first 'please' rather than the second, but that link-text is far from perfect and I haven't come up up with anything better, so no worries.

The disappearing, empty `thead` is a puzzle — I double-checked my copy of the original table and it was definitely there. Gremlins?!?
